apps/web/pages/reschedule/[uid].tsx

```diff
@@ -11,9 +12,32 @@ export default function Type() {
 }
 
 export async function getServerSideProps(context: GetServerSidePropsContext) {
+  let bookingUid = z.string().parse(context.query.uid);
+
+  // Booking uid while rescheduling can be a reference to bookingSeatsReferences Table
+  if (bookingUid) {
+    // Look bookingUid in bookingSeatsReferences
+    const bookingSeatReference = await prisma.bookingSeatsReferences.findUnique({
+      where: {
+        referenceUId: context.query.uid as string,
```

Comment:
We already parsed this

```suggestion
        referenceUId: bookingUid,
```